src/templates/Studio/Wonder/Mirrors/Mirrors.js

```diff
@@ -17,10 +17,14 @@ const Mirrors = ({ layout, scene, world }) => {
     const activeMirrorContent = [];
     const addMirrors = () => {
       layout.forEach(mirrorLayout => {
-        const mirror = MeshBuilder.CreatePlane('glass', {
-          width: mirrorLayout.width,
-          height: mirrorLayout.height,
-        }, scene);
+        const mirror = MeshBuilder.CreatePlane(
```

Comment:
Please remove unused files (Mirrors.js, and any other parts of previous scene).

Removing unused code keeps the repo cleaner and we will still be able to find them in the Git history when we need them.

Reply:
Removed